llvm/utils/TableGen/CodeEmitterGen.cpp

```diff
@@ -453,6 +447,16 @@ void CodeEmitterGen::emitCaseMap(
   }
 }
 
+static void ReportError(raw_ostream &O, indent Indent) {
```

Comment:
I'd prefer the refactoring done in a separate commit.
I have several nits that I don't want to slow down this PR.

Reply:
Fair enough, I'll revert the ReportError  refactoring.

Reply:
Done.